src/OrchardCore.Modules/OrchardCore.Contents/Workflows/Activities/CreateContentTask.cs

```diff
@@ -16,11 +17,18 @@ namespace OrchardCore.Contents.Workflows.Activities
     public class CreateContentTask : ContentTask
     {
         private readonly IWorkflowExpressionEvaluator _expressionEvaluator;
+        private readonly JavaScriptEncoder _javaScriptEncoder;
 
-        public CreateContentTask(IContentManager contentManager, IWorkflowExpressionEvaluator expressionEvaluator, IWorkflowScriptEvaluator scriptEvaluator, IStringLocalizer<CreateContentTask> localizer)
+        public CreateContentTask(
+            IContentManager contentManager,
+            IWorkflowExpressionEvaluator expressionEvaluator, 
+            IWorkflowScriptEvaluator scriptEvaluator, 
+            IStringLocalizer<CreateContentTask> localizer,
+            JavaScriptEncoder javaScriptEncoder)
```

Comment:
I wondered if this should be a `JavaScriptEncoder` or `HtmlEncoder`, or even a `NullEncoder`, as it would affect liquid values going into the database (I thought) but I tried it as it is now, and it didn't seem to be an issue.

For the rest LGTM

Reply:
We pass JSON here so the data will be javascript encoded. 
Unless we pass a HTML field which has some encoded HTML chars in it. Though nothing prevents to decode it later on. I think it's fine like it is. Maybe double check for a HTML field outcome.

Reply:
I also thought about not using any encoder in `CreateContentTask` and `UpdateContentTask` but was not sure.

Maybe what we need here is to do some `SanitizeHtml` as we do in some driver `UpdateAsync()`. If so, as we did handler `ValidateAsync()` to do part of the driver updating work in WF activities and API controllers, we would need to do the same for sanitization.

Reply:
Ok I checked what happens here.

A liquid expression like this
```
{% assign text = "België2" %}
{
  "DisplayText": "{{ text }}"
}
```

gets javascript encoded to this
```
{
  "DisplayText": "Belgi\u00EB2"
}
```

but `JSON` unecodes it (presumably because it respects javascript encoding), so after a `JObject.Parse` it becomes
```
{
  "DisplayText": "België2"
}
```

So what gets saved in the database is ultimately correct.

In which case why bother encoding it in the first place maybe?

For info @jtkech `SanitizeHtml` with the sanitizer is difficult to do on individual properties (possible on a property as a whole), but it was felt that `Workflows` was out of scope - as someone with an edit `Workflow` permission can do so many things already

Reply:
Why bother? Try with `België2"` and without encoding it would break the json.

Reply:
I am merging, but it doesn't mean we can't change it if we think it's wrong.